pymc4/tests/test_template_context.py

```diff
@@ -27,7 +28,7 @@ def test_free_forward_context_add_variable(monkeypatch):
 
 
 def test_free_forward_context_var_as_backend_tensor(monkeypatch):
-    """Test that random variable initializes with in Free Forward context"""
+    """Test that random variable initialize in Free Forward context"""
```

Comment:
Suggested grammatical change:

```
    """Test that random variable initializes in Free Forward context"""
```

Reply:
Let me know if you think the new docstring is good